.github/workflows/job-demand.yml

```diff
@@ -0,0 +1,35 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  validate:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Install JavaScript dependencies
+      - name: Install
+        uses: CultureHQ/actions-yarn@master
+        with:
+          args: install
+
+      # Validate current working state of project
+      - name: Validate
+        uses: CultureHQ/actions-yarn@v1.0.1
+        with:
+          args: validate
```

Comment:
This is all we need to configure CI with GIthub Actions :). Notice the `uses: CultureHQ/actions-yarn@1.0.1` which encapsulates much of the configuration in a published action for runnning `yarn` commands.